### PR TITLE
SPP-9546-amend-IDS-link-description

### PR DESCRIPTION
**Pull request by joepercival**
# Description

Links have been presented on the page using URLs which do not describe the purpose of the links without the additional surrounding context on the page. This PR will amend the descriptions of these links in order to make them more accessible to users navigating out of the context of the page.

Code has not been commented and documentation has not been provided because it is not necessary for this PR.

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Technical Enhancements

# Checklist:

If any of these are not completed, please explain why in the notes.

## **Definition of Done**
**Code and merges**

- [ ] Code to be commented on where applicable 
- [ ] Documentation updated where required 
- [x] Have considered non-functional requirements such as Security, Performance, Scalability, and Fault Tolerance
- [x] I have linted the code

**Testing**

- [x] All levels of acceptance test are passing (automated, integration, manual, accessibility, etc.)
- [x] I have run the behave command to check the selenium behaviour tests pass locally
- [x] Acceptance criteria met

**Other Checks**
- [x] I have performed a self-review of my own code
- [x] I have checked for spelling errors
- [ ] I have commented on my code, particularly in hard-to-understand areas
- [x] My changes don't break anything unexpected
- [x] I have checked and updated the security.txt file where required
- [x] Up to date with the main branch

### CODE REVIEW

**Comment by ons-spp-machine-user** [comment]
DEPLOY_URL=https://drnd6r2akxm01.cloudfront.net/
cloudfront_id=E1VWPRF2OIE3K8

**Comment by ons-spp-machine-user** [comment]
DEPLOY_URL=https://drnd6r2akxm01.cloudfront.net/
cloudfront_id=E1VWPRF2OIE3K8

File: content/help_centre/run-a-method.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 1. Go to the GitHub repository page (linked to from method's details page in the catalogue), and click on "Releases". Find the release you want, noting any warnings or information about that version.
-Release pages for both repositories can be found below:  
-[https://github.com/ONSdigital/sml-python-small/releases](https://github.com/ONSdigital/sml-python-small/releases)
-[https://github.com/ONSdigital/statistical-methods-library/releases](https://github.com/ONSdigital/statistical-methods-library/releases)
+Release pages for both repositories can be found below:
+    - [SML Python-Small releases](https://github.com/ONSdigital/sml-python-small/releases)
```
Comment (jhmjam) [inline content/help_centre/run-a-method.md:3]: May want to remove the dash here and add repository to the end of small :)

**Review by jhmjam — CHANGES_REQUESTED** [review]

**Review by jhmjam — DISMISSED** [review]

**Comment by ons-spp-machine-user** [comment]
DEPLOY_URL=https://drnd6r2akxm01.cloudfront.net/
cloudfront_id=E1VWPRF2OIE3K8

**Review by jhmjam — APPROVED** [review]

**Comment by ons-spp-machine-user** [comment]
DEPLOY_URL=https://drnd6r2akxm01.cloudfront.net/
cloudfront_id=E1VWPRF2OIE3K8

**Pull request merged**